analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -109,6 +109,7 @@ public void shutdown() {
     private ExecutorService networkExecutor;
     private ThreadFactory threadFactory;
     private int flushQueueSize;
+    private int maximumFlushAttempts;
```

Comment:
Is there a reason the naming of this is maximumFlushAttempts? Any issues if we refactor this and keep consistent with "maxRetries"?

Reply:
yea, that makes sense!